EKF/control.cpp

```diff
@@ -243,16 +256,57 @@ void Ekf::controlExternalVisionFusion()
 			_ev_sample_delayed.posNED(1) -= pos_offset_earth(1);
 			_ev_sample_delayed.posNED(2) -= pos_offset_earth(2);
 
-			// if GPS data is being used, then use an incremental position fusion method for EV data
-			if (_control_status.flags.gps) {
-				_hpos_odometry = true;
+			// Use an incremental position fusion method for EV data if using GPS or if the observations are not in NED
+			if (_control_status.flags.gps || (_params.fusion_mode & MASK_ROTATE_EV)) {
+				_fuse_hpos_as_odom = true;
```

Comment:
I believe that this makes https://github.com/PX4/ecl/pull/312/files#diff-4cac31c333867233bb8a08de1e812a9aR171 redundant.

Reply:
I think you are right